src/components/atoms/Reactions/index.tsx

```diff
@@ -0,0 +1,203 @@
+import React from "react";
+import { styled } from "@pigment-css/react";
+import Box from "../Box";
+import { NeynarAuthButton } from "../../organisms/NeynarAuthButton";
+import { LocalStorageKeys } from "../../../hooks/use-local-storage-state";
+import { NEYNAR_API_URL } from "../../../constants";
+import { useNeynarContext } from "../../../contexts";
+import { SIWN_variant } from "../../../enums";
+import { CommentIcon } from "../icons/CommentIcon";
+import { RecastIcon } from "../icons/RecastIcon";
+import { LikeIcon } from "../icons/LikeIcon";
+import XIcon from "../icons/XIcon";
+
+const ReactionWrapper = styled(Box)(({ theme }) => ({
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center",
+  position: "relative",
+})) as typeof Box;
+
+const Popover = styled(Box)(({ theme }) => ({
+  position: "absolute",
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  backgroundColor: theme?.colors?.background || "#fff",
+  padding: '5px 8px',
+  boxShadow: theme?.shadows?.[2] || "0px 4px 6px rgba(0, 0, 0, 0.1)",
+  zIndex: theme?.zIndex?.popover || 2000,
+  borderRadius: 4,
+  minWidth: '245px',
+  width: 'auto',
+  maxWidth: '90vw',
+})) as typeof Box;
+
+const PopoverContent = styled(Box)({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  flex: 1,
+}) as typeof Box;
+
+const CloseButtonWrapper = styled(Box)({
+  display: 'flex',
+  alignItems: 'center',
+  marginLeft: '8px',
+}) as typeof Box;
+
+type ReactionsProps = {
+  hash: string;
+  onComment?: () => void;
+  onRecast?: () => void;
+  onLike?: () => void;
+};
+
+const Reactions: React.FC<ReactionsProps> = ({
```

Comment:
@dylsteck 
No need to change this now, we can ship it with it.
Reactions should be a molecule-level component (idea being multiple atoms --reactions in this case-- form a molecule)

Reply:
ok gotcha, for the next round of edits in the future I'll make `Reactions` a molecule and the individual reactions themselves atoms